src/api/server-routes.ts

```diff
@@ -294,6 +297,74 @@ export function repositoryRoutes(): express.Router {
     }).catch(err => res.status(401).json({ data: 'Not Authorized' }));
   });
 
+  router.get('/check/:id', (req: express.Request, res: express.Response) => {
+    let repository: Repository = null;
+    getRepository(req.params.id).then(repo => {
+      let accessToken = null;
+      if (repo.access_token && repo.access_token[0]) {
+        accessToken = repo.access_token[0].token || null;
+      }
+      repository = {
+        clone_url: repo.clone_url,
+        branch: 'master',
```

Comment:
you could use repo.default_branch here, it's probably better...

Reply:
fixed. thanks